schemas/hpc-job-scheduler.md

```diff
@@ -0,0 +1,41 @@
+# HPC Job Scheduler Event Schema
+
+NOTE: Work in Progress!
+
+Tools being targeted:
+
+- Slurm Job Logs
+- Flux Job Logs
+
+## Job Log Schema
+
+| **Field Name**         | **Field Explanation**                                                                | **Slurm**       | **Flux**         |
+| ---------------------- | ------------------------------------------------------------------------------------ | --------------- | ---------------- |
+| job.id                 | Unique identifier of the job                                                         | JobId           | id               |
+| user.name              | User name of who submitted the job                                                   | UserId          | username         |
+| user.id                | User ID number of who submitted the job                                              | UserId          | userId           |
+| group.id               | Group ID number of who submitted the job                                             | GroupId         |                  |
+| group.name             | Group ID number of who submitted the job                                             | GroupId         | jobspec.name     |
+| event.outcome          | The final state of the job                                                           | JobState        | result           |
+| job.queue              | The queue the job was submitted to                                                   | Partition       | queue            |
+| job.timelimit          | Maximum time job could have ran for.                                                 | TimeLimit       | expiration       |
+| event.start            | Time that the job started.                                                           | StartTime       | t_run            |
+| event.end              | Time that the job ended.                                                             | EndTime         | t_inactive       |
+| event.duration         | The actual amount of time the job ran for (event.start - event.end) (in nanoseconds) |                 | jobspec.duration |
+| event.duration_seconds | The actual amount of time the job ran for (event.start - event.end) (in seconds)     |                 |                  |
+| job.node.list          | list of nodes being used, can be a range ex: corona[155, 157, 180-190]               | NodeList        | R.hostlist       |
+| job.node.count         | Count of nodes used                                                                  | NodeCnt         | nnodes           |
+| job.proc.count         | Number of processors (typically number of processes per node \* number of nodes)     | ProcCnt         |                  |
```

Comment:
FWIW, i am calculating this manually in my job record script, so maybe for the Flux column we can put: `nnodes` * `ntasks`?